[CS2113-T15-2] SecureNUS

ollayf opened the pull request:
SecureNUS is a desktop CLI app for managing and storing passwords. SecureNUS provides the functionalities of a regular password manager tool, but on CLI. Simple and efficient to use especially for fast typists.

ysl-28 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,442 @@
+
+The Sequence Diagram below shows how the components interact with each other for the scenario where the user creates a new basic password initiated using the command `new password.`
+
+Sequence Diagram
```
It might not be too clear that methods like `extractName()` and `inquireURL()` are only being called in `AddBasicPassword` and not in the other classes that inherit from `Command`.

ysl-28 replied:
For methods like `inquireURL()` that return a string, perhaps the return arrows should be labelled with that they are returning?

ysl-28 replied:
Might it be clearer to show object deletion at the end of its lifeline too?

ysl-28 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,442 @@
+
+<img src="./DGDiagramsCreator/DGUsedDiagrams/CommandComponent.png" width="100%" />
+
+The Command consist of Command abstract class that handles all of its command constructors and executions through its child classes. The user inputs command in Ui, that is parsed in Parser, and then institiated in and then executed in Command classes.
```
Typo for instantiated.

Thunderdragon221 commented on docs/DGDiagramsCreator/UI.puml:
```diff
@@ -0,0 +1,17 @@
+!include Style.puml
+title UI Component Diagram
+actor User
+node SecureNUS
```
Maybe represent the classes using the standard representation adopted by CS2113 instead of 3-dimensional boxes?

Thunderdragon221 replied:
Maybe represent the classes using the standard representation adopted by CS2113 instead of 3-dimensional boxes?

Thunderdragon221 commented on docs/DGDiagramsCreator/ClassHigh.puml:
```diff
@@ -0,0 +1,34 @@
+    frame Backend {
+        node Backend
+    }
+    Commands --> SecureNUS.SecureNUS
```
Maybe include details as part of the arrows for additional clarity?

stevenantya committed:
checkstyle2

Thunderdragon221 commented on docs/UML/NewBasicPassword.puml:
```diff
@@ -0,0 +1,37 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+
+autonumber
```
Maybe omit autonumbering to follow the standard practice adopted by CS2113?

Thunderdragon221 commented on docs/DGDiagramsCreator/SecretStorage.puml:
```diff
@@ -0,0 +1,14 @@
+    SecretMaster -[dotted]-> SecretSearcher: alters
+}
+() SecureNUS -[dotted]right-> SecretMaster :requests alteration \n of data via
+SecretMaster <-[dotted]right-> Backend : sends data for export OR\n collects data for import
```
I like the clarity described by the arrows in the diagram.

Thunderdragon221 reviewed (commented):
Overall good DG with minor formatting errors.

ysl-28 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,442 @@
+<img src="./DGDiagramsCreator/DGUsedDiagrams/ArchitectureDiagram.png" width="100%" />
+
+
+This **Architecture Diagram** explains the high-level design of the App.
```
Perhaps use the standard CS2113 representation for diagrams? (instead of using the symbols)

ysl-28 reviewed (commented):
DG is generally clear and understandable

kaceycsn commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,442 @@
+# Product Scope
+
+
+## Target user profile:
+
+
+
+* has a need to manage a significant number of passwords
+* prefer desktop apps over other types
+* can type fast
+* prefers typing to mouse interactions
+* is reasonably comfortable using CLI apps
+
```
May consider being more specific when defining the target users, and elaborate how the value proposition matches the needs of the target users.

kaceycsn commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,442 @@
+* is reasonably comfortable using CLI apps
+
+
+## Value proposition:
```
May consider defining the scope more clearly by specifying the boundary beyond which the app will not help.

kaceycsn commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,442 @@
+
+This is the main component that initialises all other components and connects them when the application is running.
+
+## SecretStorage Component
```
I like how the system is divided into the separate components in the documentation. The component diagrams are clear and simple.

kaceycsn commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,442 @@
+
+The Sequence Diagram below shows how the components interact with each other for the scenario where the user creates a new basic password initiated using the command `new password.`
+
+Sequence Diagram
```
In the sequence diagram, a semicolon should be added before the class name. Unnamed instances of a class should be denoted as `:Class`.

ollayf committed:
trying to resolve inf loop: 1

ollayf committed:
Tested all checks work locally

ollayf committed:
Trying to fix windows build

ollayf committed:
Merge pull request #82 from ollayf/master

trying to resolve inf loop: 1

ollayf committed:
Merge pull request #83 from AY2223S2-CS2113-T15-2/fields

Remove infinite loop in github tests

kairuler committed:
Checkstyle fixes (incl. remove ExitCommandTest)

ollayf committed:
Fixed IO Redirection issues

ollayf committed:
Merge branch 'AY2223S2-CS2113-T15-2:master' into master

ollayf committed:
Updated inputs

ollayf committed:
Added checkstyle for readers

kairuler committed:
Merge remote-tracking branch 'origin/master' into searching

# Conflicts:
#	src/test/java/seedu/duke/command/AddCommandTest.java
#	src/test/java/seedu/duke/command/DeleteCommandTest.java

ollayf committed:
Re add actual txt

ollayf committed:
altered the file structure for bat file

kairuler committed:
Merge remote-tracking branch 'upstream/export' into searching

# Conflicts:
#	src/main/java/seedu/duke/Backend.java
#	src/main/java/seedu/duke/Parser.java
#	src/main/java/seedu/duke/exceptions/secrets/FolderExistsException.java
#	src/main/java/seedu/duke/secrets/CreditCard.java
#	src/test/java/seedu/duke/DukeTest.java

kairuler committed:
Merge branch 'master' of https://github.com/kairuler/tp into searching

# Conflicts:
#	src/main/java/seedu/duke/Backend.java
#	src/main/java/seedu/duke/command/ListCommand.java
#	src/main/java/seedu/duke/exceptions/secrets/FolderExistsException.java
#	src/main/java/seedu/duke/secrets/CreditCard.java
#	src/test/java/seedu/duke/DukeTest.java
#	src/test/java/seedu/duke/command/ExitCommandTest.java

kairuler committed:
Merge pull request #3 from kairuler/searching

Checkstyle fixes (incl. remove ExitCommandTest)

kairuler committed:
v2.0 release

kairuler committed:
Merge pull request #4 from kairuler/searching

v2.0 release

kairuler committed:
Merge branch 'master' into master

kairuler committed:
Merge pull request #86 from kairuler/master

v2.0 release

stevenantya committed:
Merge pull request #81 from stevenantya/v1.31

Fix bugs

stevenantya committed:
Merge branch 'master' into creation

ollayf committed:
Merging for v2.0

stevenantya committed:
khairul gradlew

aaronxujiachen commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,442 @@
+
+
+1. **Configure the coding style** \
+   If using IDEA, follow the guide [[se-edu/guides] IDEA: Configuring the code style](https://se-education.org/guides/tutorials/intellijCodeStyle.html) to set up IDEA’s coding style to match ours. 
```
I am very glad that all the hyperlinks are working. The contents shown in the links are also sufficient enough for readers to understand the author's intention.

aaronxujiachen commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,442 @@
+
+
+
+1. **Configure the JDK:** Follow the guide [[se-edu/guides] IDEA: Configuring the JDK](https://se-education.org/guides/tutorials/intellijJdk.html) to ensure Intellij is configured to use JDK 11.
+2. **Import the project as a Gradle project:** Follow the guide [[se-edu/guides] IDEA: Importing a Gradle project](https://se-education.org/guides/tutorials/intellijImportGradleProject.html) to import the project into IDEA. \
+
+<img src="./DGDiagramsCreator/DGUsedDiagrams/ExclamationMark.png" width="4%" />
+Note: Importing a Gradle project is slightly different from importing a normal Java project.
+3. **Verify the setup:**
+    1. Run the _seedu.address.Main_ and try a few commands.
+    2. [Run the tests](https://se-education.org/addressbook-level3/Testing.html) to ensure they all pass.
```
Perhaps the indentation can be done better? So that it will be more reader friendly.

shannentan replied:
Are these numberings supposed to have been changed?
![image](https://user-images.githubusercontent.com/86601208/228775574-227edc42-a8ce-4bb2-b696-1d20c39247dd.png)

aaronxujiachen commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,442 @@
+
+## Architecture
+
+<img src="./DGDiagramsCreator/DGUsedDiagrams/ArchitectureDiagram.png" width="100%" />
```
Very clear and detailed diagram.

aaronxujiachen commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,442 @@
+---
+
+[//]: # (@@author kairuler)
+# User Stories
```
The user stories are elaborated and show a good progression of the product development.

shannentan commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,442 @@
+
+The UI consists of the Ui class which handles all the inputs provided by the user. The Ui then sends information to the 
+parser which will interpret the commands provided by user input. The **Parser **will create **Commands** which will be
+sent to **SecureNUS** to execute some command that will alter the data in SecretStorage. The **Ui **also handles 
```
What are the ** for? Are there more than necessary?
![image](https://user-images.githubusercontent.com/86601208/228768749-5b5399c3-94e9-4272-a5f0-0c719fc323ad.png)

shannentan commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,442 @@
+* Should be user-friendly, with intuitive commands that makes it easy for users to create, store, and retrieve passwords.
+* Should be compatible with various operating systems as long as Java 11 or above is installed.
+* Should be reliable and operational all the time with little-to-no bugs or errors.
+* Should be able to store at least 1000 passwords without compromising performance or security.
```
Is the word 'store' missing on the page?
![image](https://user-images.githubusercontent.com/86601208/228770458-65e6050f-518d-4914-b3c3-e5b3cc979c6c.png)

shannentan commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,442 @@
+
+## SecretStorage Component
+
+<img src="./DGDiagramsCreator/DGUsedDiagrams/SecretStorageComponent.png" width="100%" />
```
Perhaps it would be easier to read if the words 'requests alteration of data via'  and 'sends data for export OR collects data for import' were inside the box instead? (Such that there won't be lines cutting through the words)

shannentan commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,442 @@
+<img src="./DGDiagramsCreator/DGUsedDiagrams/ExclamationMark.png" width="4%" />
+Tip: Optionally, you can follow the guide [[se-edu/guides] Using Checkstyle](https://se-education.org/guides/tutorials/checkstyle.html) to find how to use the CheckStyle within IDEA e.g., to report problems as you write code.  
+
+2. **Set up CI** This project comes with a GitHub Actions config files (in .github/workflows folder). When GitHub detects those files, it will run the CI for your project automatically at each push to the master branch or to any PR. No set up required.  
+3. **Learn the design** When you are ready to start coding, we recommend that you get some sense of the overall design by reading about [AddressBook’s architecture](https://se-education.org/addressbook-level3/DeveloperGuide.html#architecture).  
+4. **Do the tutorials These tutorials will help you get acquainted with the codebase.**  
```
Are these numberings updated in the page?
![image](https://user-images.githubusercontent.com/86601208/228775319-a71168d5-16f0-4d80-9604-88126ee07848.png)

ollayf committed:
Update EditCommandTest.java

ollayf committed:
Update EditCommandTest.java

stevenantya committed:
Contribute to DG

stevenantya committed:
Merge pull request #93 from stevenantya/master

Add UG

stevenantya committed:
Add UG

Add UG

euzhengxi committed:
added formatting for UG

euzhengxi committed:
Merge pull request #95 from euzhengxi/export

Export: added minor formatting changes to the UG

euzhengxi committed:
Merge pull request #96 from AY2223S2-CS2113-T15-2/export

Export

DeepanjaliDhawan committed:
Updates

DeepanjaliDhawan committed:
Revert "Updates"

This reverts commit 8e3a22f4ccd2218a01816c48bf0f6746955e4dde.

DeepanjaliDhawan committed:
Merge remote-tracking branch 'upstream/master'

kairuler committed:
Fix Edit command to change all fields for BasicPassword, CreditCard, NUSNet, StudentID

kairuler committed:
Small fix for Edit command

kairuler committed:
Merge pull request #5 from kairuler/searching

Fix Edit command to actually change password

kairuler committed:
Fix JUnit

kairuler committed:
Merge pull request #6 from kairuler/searching

Fix JUnit

kairuler committed:
Added Edit support for Wifi Password and Crypto Wallet

kairuler committed:
Merge pull request #7 from kairuler/searching

Added Edit support for Wifi Password and Crypto Wallet

ollayf committed:
Revamped Add Commands System

ollayf committed:
Edited the text ui test

ollayf committed:
Edited text ui test

ollayf committed:
Merge pull request #171 from kairuler/master

Fix Edit command and added JUnit tests

DeepanjaliDhawan committed:
Commits:
1. Update List
2. Update a layout for UG
3. Update PPP
4. Update AboutUS

DeepanjaliDhawan committed:
correct checkstyle error

DeepanjaliDhawan committed:
Update the expected.txt

DeepanjaliDhawan committed:
Update2

DeepanjaliDhawan committed:
update3

DeepanjaliDhawan committed:
Final Update

DeepanjaliDhawan committed:
Update- expected.txt

DeepanjaliDhawan committed:
Merge pull request #175 from DeepanjaliDhawan/master

Updates

DeepanjaliDhawan committed:
Merge branch 'master' into tagging

DeepanjaliDhawan committed:
Merge branch 'master' into master

DeepanjaliDhawan committed:
Solve errors

DeepanjaliDhawan committed:
Merge pull request #179 from DeepanjaliDhawan/master

Latest Updates - Nusnet ID

stevenantya committed:
Fix delete multiple items

stevenantya committed:
add uml

stevenantya committed:
Merge pull request #180 from stevenantya/v2.10

Fix delete multiples and add UML V2.10

stevenantya committed:
Merge pull request #181 from AY2223S2-CS2113-T15-2/creation

Creation V2.1

ollayf committed:
Merge pull request #178 from AY2223S2-CS2113-T15-2/tagging

Tagging

ollayf committed:
Merged all functions except edit and search

ollayf committed:
Merge pull request #182 from ollayf/master

Checked everything except search and edit

kairuler committed:
Draft PPP

kairuler committed:
Merge pull request #183 from kairuler/master

Draft PPP

DeepanjaliDhawan committed:
Update PPP

DeepanjaliDhawan committed:
Merge pull request #184 from DeepanjaliDhawan/master

Update PPP

ollayf committed:
Merged Search and View Function

ollayf committed:
Merged all main features

ollayf committed:
Remove trimming for JUnit tests

ollayf committed:
Edited reveal strings

ollayf committed:
Removed tests that gave me some issues from viewcommand

ollayf committed:
Removed imports causing issues

ollayf committed:
Merge pull request #185 from ollayf/master

Merged all main features

kairuler committed:
Fix JUnit tests for Edit and View
Also fixed minor issues in Edit, Search, View commands

kairuler committed:
Merge pull request #186 from kairuler/master

Fix JUnit tests for Edit and View

ollayf committed:
Updated the menu and starting commands

ollayf committed:
Updated menus and starting messages

ollayf committed:
edited text ui tests

ollayf committed:
removed additional stuff

euzhengxi committed:
fixed bugs in backend, added save command, added assertions

ollayf committed:
Merge pull request #187 from ollayf/master

Edited UI including printing and formatting of most functions and menu and list functions

ollayf committed:
Fixed the case where folders cannot be empty

ollayf committed:
Changed criteria for legal secretnames and fodlers

ollayf committed:
Merge branch 'fixing_empty_input_issues'

euzhengxi committed:
resolveg merge conflicts

ollayf committed:
Merge pull request #189 from ollayf/master

Fixed problem with empty strings for secret names and folders

euzhengxi committed:
made amendments according to checkstyle error

euzhengxi committed:
made amendments to test

euzhengxi committed:
amended text ui test

euzhengxi committed:
Merge pull request #188 from euzhengxi/export

amended bugs in backend, added save command, assertions and made amendments to correct checkstyle, test and text ui tests

euzhengxi committed:
amended bugs in backend

ollayf committed:
Merge branch 'master' into export

ollayf committed:
Update SecretMaster.java

ollayf committed:
Merge pull request #190 from AY2223S2-CS2113-T15-2/export

Export

euzhengxi committed:
Merge pull request #191 from euzhengxi/export

Export

euzhengxi committed:
Merge pull request #192 from AY2223S2-CS2113-T15-2/export

Export - amended bugs in backend: corrupted data will not be loaded into secretmaster

kairuler committed:
Minor edits

euzhengxi committed:
fixed text-ui-test

kairuler committed:
Merge pull request #218 from kairuler/master

Minor edits

kairuler committed:
Make Search command output more visually appealing
Also update JUnit tests for Search command

kairuler committed:
Merge pull request #222 from kairuler/master

Make Search command output more visually appealing

euzhengxi committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-T15-2/tp into export

euzhengxi committed:
Added details for PPP draft

euzhengxi committed:
amended text-ui-tests

euzhengxi committed:
Merge pull request #223 from euzhengxi/export

Export - added details for PPP draft

euzhengxi committed:
Merge pull request #224 from AY2223S2-CS2113-T15-2/export

Export - added details for PPP draft

euzhengxi committed:
made amendments according to team testing

euzhengxi committed:
Merge pull request #225 from euzhengxi/export

Export - Made amendments according to team testing

ollayf committed:
Main changes from pentesting and issues flagged out in github

ollayf committed:
Cleared all issues after PED code related

ollayf committed:
Best release for testing round 2

ollayf committed:
Removed database.txt file after every test

ollayf committed:
Ready for testing round 2 without database coverage

ollayf committed:
Merge pull request #227 from ollayf/master

Cleared issues post PED and team test 1

DeepanjaliDhawan committed:
Update PPP with required naming convention

DeepanjaliDhawan committed:
Merge pull request #228 from DeepanjaliDhawan/deeps-ppp

Deeps ppp

DeepanjaliDhawan committed:
Merge pull request #229 from AY2223S2-CS2113-T15-2/tagging

PPP Naming convention

ollayf committed:
Updated instructions for users in menu adn startup

ollayf committed:
Merge pull request #230 from ollayf/master

Updated instructions for users in menu adn startup

stevenantya committed:
Add colon in sequence diagram to represent class

stevenantya committed:
Fix issue #165.

Fix issue #165. Sequence diagram object representation

euzhengxi committed:
added information to print to user when session starts

kairuler committed:
Add JUnit test
Aim is to get Method coverage >80%

euzhengxi committed:
amended backend bugs and implemented autosave function

euzhengxi committed:
Merge branch 'export' into export

euzhengxi committed:
Update Ui.java

euzhengxi committed:
Update AddSecretCommand.java

euzhengxi committed:
Update DeleteCommand.java

euzhengxi committed:
Update Backend.java

euzhengxi committed:
Fixed bugs in backend - reading and writing special characters in database

euzhengxi committed:
resolved merge conflicts

euzhengxi committed:
resolved bugs in exceptions

euzhengxi committed:
made amendments to text-ui-test

euzhengxi committed:
amended junit tests

ollayf committed:
Refactored naming

ollayf committed:
Update OperationMessages.java

ollayf committed:
Merge pull request #235 from euzhengxi/export

Export

ollayf committed:
Merge pull request #226 from AY2223S2-CS2113-T15-2/export

Export

ollayf committed:
Prepping for release all segments merged

ollayf committed:
Merge pull request #236 from ollayf/master

Prep for final release, merged all main sections

ollayf committed:
Edited runtests

ollayf committed:
Added all the new diagrams from Hosea

ollayf committed:
Merge pull request #237 from ollayf/master

Added documentation for DG

DeepanjaliDhawan committed:
Updates
1. UG
2. ReadMe
3. Javadocs

DeepanjaliDhawan committed:
Merge pull request #238 from DeepanjaliDhawan/master

Updates to UG

stevenantya committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-T15-2/tp

DeepanjaliDhawan committed:
Merge pull request #239 from AY2223S2-CS2113-T15-2/tagging

Updates in UG

euzhengxi committed:
added more details for PPP and logging

DeepanjaliDhawan committed:
Updates in Add

euzhengxi committed:
pulled from upstream

DeepanjaliDhawan committed:
Merge pull request #240 from DeepanjaliDhawan/master

Changes in UG add

DeepanjaliDhawan committed:
Make changes in Add

DeepanjaliDhawan committed:
Merge pull request #241 from DeepanjaliDhawan/master

Make changes in Add

DeepanjaliDhawan committed:
Update UserGuide.md

DeepanjaliDhawan committed:
Updates

DeepanjaliDhawan committed:
Merge pull request #242 from DeepanjaliDhawan/master

Updates

DeepanjaliDhawan committed:
Updates

DeepanjaliDhawan committed:
Merge pull request #243 from DeepanjaliDhawan/master

Updates

DeepanjaliDhawan committed:
Updates to UG

DeepanjaliDhawan committed:
UG updates

DeepanjaliDhawan committed:
Merge pull request #244 from DeepanjaliDhawan/master

Updating UG

DeepanjaliDhawan committed:
Testing update in UG

DeepanjaliDhawan committed:
Merge pull request #245 from DeepanjaliDhawan/master

Testing update in UG

DeepanjaliDhawan committed:
Formatting updates in UG

DeepanjaliDhawan committed:
Merge pull request #246 from DeepanjaliDhawan/master

Formatting updates in UG

DeepanjaliDhawan committed:
Update UserGuide.md

DeepanjaliDhawan committed:
Update PPP link

DeepanjaliDhawan committed:
Update AboutUs.md

ollayf committed:
Edited annoying saving messages

ollayf committed:
Ready for v2.1 final testing

ollayf committed:
Merge pull request #247 from ollayf/master

Ready for final testing before v2.1

DeepanjaliDhawan committed:
Merge branch 'master' of https://github.com/DeepanjaliDhawan/tp

DeepanjaliDhawan committed:
Modifications to UG

kairuler committed:
Merge remote-tracking branch 'upstream/master' into test

# Conflicts:
#	src/main/java/seedu/duke/command/Command.java
#	src/main/java/seedu/duke/command/ViewCommand.java
#	src/main/java/seedu/duke/ui/Parser.java
#	src/main/java/seedu/duke/ui/Ui.java
#	src/main/java/seedu/securenus/command/AddCreditCardCommand.java
#	src/test/java/seedu/duke/BackendTest.java
#	src/test/java/seedu/duke/SecureNUSTest.java
#	src/test/java/seedu/securenus/SecureNUSLogFormatterTest.java
#	src/test/java/seedu/securenus/SecureNUSLoggerTest.java
#	src/test/java/seedu/securenus/command/AddBasicPasswordCommandTest.java
#	src/test/java/seedu/securenus/command/AddCommandTest.java
#	src/test/java/seedu/securenus/command/AddCreditCardCommandTest.java
#	src/test/java/seedu/securenus/command/AddCryptoWalletCommandTest.java
#	src/test/java/seedu/securenus/command/AddNUSNetCommandTest.java
#	src/test/java/seedu/securenus/command/AddSecretCommandTest.java
#	src/test/java/seedu/securenus/command/AddStudentIDCommandTest.java
#	src/test/java/seedu/securenus/command/AddWifiPasswordCommandTest.java
#	src/test/java/seedu/securenus/command/CommandTest.java
#	src/test/java/seedu/securenus/ui/ParserTest.java
#	src/test/java/seedu/securenus/ui/UiTest.java

kairuler committed:
Updated JUnit tests in accordance with Backend changes made by zhengxi

kairuler committed:
Edit AboutUs

kairuler committed:
Edit AboutUs

kairuler committed:
Merge pull request #233 from kairuler/master

Add JUnit test

euzhengxi committed:
added logging capabilities to SecureNUS

euzhengxi committed:
fixed checkstyle errors

euzhengxi committed:
Merge pull request #248 from euzhengxi/export

Export

euzhengxi committed:
Merge branch 'master' into export

euzhengxi committed:
Update AddCreditCardCommand.java

euzhengxi committed:
Update SecretMaster.java

euzhengxi committed:
fixed merge conflicts and junit test for logging

euzhengxi committed:
Merge branch 'export' into export

euzhengxi committed:
Merge pull request #250 from euzhengxi/export

fixed merge conflicts and junit test for logging

euzhengxi committed:
Merge pull request #249 from AY2223S2-CS2113-T15-2/export

Export - fixed merge conflicts and junit test for logging

DeepanjaliDhawan committed:
Minor fixes in UG

DeepanjaliDhawan committed:
Merge pull request #257 from DeepanjaliDhawan/master

Updates in UG

kairuler committed:
Finalise JUnit tests
Remove JUnit tests involving Scanner
Remove overloaded methods used solely for JUnit tests involving Scanner

kairuler committed:
Merge pull request #258 from kairuler/master

Finalise JUnit tests

kairuler committed:
Clean up PPP

stevenantya committed:
Add and fix DG

stevenantya committed:
Merge pull request #260 from stevenantya/v2.1a

V2.1a

stevenantya committed:
Fix and update DG UMLs

Fix DG, Fix issue #165.

DeepanjaliDhawan committed:
Update PPP

kairuler committed:
Merge pull request #259 from kairuler/master

Clean up PPP

stevenantya committed:
Add DG images

kairuler committed:
Minor bug in AboutUs.md

kairuler committed:
Merge pull request #261 from kairuler/master

Minor bug in AboutUs.md

DeepanjaliDhawan committed:
Minor edits to UG

DeepanjaliDhawan committed:
Merge pull request #262 from DeepanjaliDhawan/master

Minor edits to UG

DeepanjaliDhawan committed:
Update UserGuide.md

stevenantya committed:
Update DG Final

stevenantya committed:
Fix minor DG bugs

stevenantya committed:
Merge pull request #263 from stevenantya/v2.1b

V2.1b Add Newest DG

stevenantya committed:
Merge pull request #264 from AY2223S2-CS2113-T15-2/creation

Add final DG

ollayf committed:
Removed excess methods for testing

ollayf committed:
Added links in md

ollayf committed:
Merge pull request #265 from ollayf/master

Links to headers

stevenantya committed:
Fix bugs

stevenantya committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-T15-2/tp

ollayf committed:
Removed additional links

ollayf committed:
Added last changes in DG

ollayf committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-T15-2/tp

ollayf committed:
Final changes to dg

ollayf committed:
Fixed bolding

DeepanjaliDhawan committed:
Final edits to edit command

ollayf committed:
final fixes

ollayf committed:
arrows

ollayf committed:
escape triangle braces

DeepanjaliDhawan committed:
Merge pull request #266 from DeepanjaliDhawan/master

Final edits to edit command

ollayf committed:
resized images

ollayf committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-T15-2/tp

DeepanjaliDhawan committed:
Updates

ollayf committed:
Merge pull request #267 from ollayf/master

Resized images

DeepanjaliDhawan committed:
updates

ollayf committed:
removed additional backslshes

ollayf committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-T15-2/tp

DeepanjaliDhawan committed:
Merge pull request #268 from DeepanjaliDhawan/master

Updates

ollayf committed:
Resize images and center

ollayf committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-T15-2/tp

DeepanjaliDhawan committed:
Center images

DeepanjaliDhawan committed:
Merge pull request #269 from DeepanjaliDhawan/master

Center images

ollayf committed:
minor edits

ollayf committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-T15-2/tp

kairuler committed:
Finalise PPP

kairuler committed:
Merge pull request #8 from kairuler/test

Finalise PPP

kairuler committed:
Merge pull request #270 from kairuler/master

Finalise PPP

DeepanjaliDhawan committed:
Convert current Jarfile name to SecureNUS.jar

DeepanjaliDhawan committed:
Merge pull request #271 from DeepanjaliDhawan/master

Convert current Jarfile name to SecureNUS.jar

ollayf committed:
Edited PP

ollayf committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-T15-2/tp

ollayf committed:
Removed log file

stevenantya committed:
fix steven naming ppp
